docs/configuration.md

```diff
@@ -179,6 +179,15 @@ of the most common options to set are:
     (e.g. <code>2g</code>, <code>8g</code>).
   </td>
 </tr>
+<tr>
+ <td><code>spark.executor.pyspark.memory</code></td>
+  <td>Not set</td>
+  <td>
+    The amount of memory to be allocated to PySpark in each executor, in MiB
+    unless otherwise specified.  If set, PySpark memory for an executor will be
+    limited to this amount. If not set, Spark will not limit Python's memory use.
```

Comment:
Maybe mention that in this case (unset) it's up to the user to keep Python + system processes in the overhead %.

Reply:
I've added "and it is up to the application to avoid exceeding the overhead memory space shared with other non-JVM processes."